packages/react/src/experimental/OneDataCollection/visualizations/collection/Table/components/Row.tsx

```diff
@@ -147,13 +197,35 @@ const RowComponentInner = <
           sticky={{
```

Comment:
Makes no sense to show the table cell as the actions are now only shown on hover (only makes sense to have then in the mobile version for the fixed dropdown